paver.conf

```diff
@@ -1,4 +1,4 @@
-BASE_REQUIREMENTS='chevah-brink==0.63.3 paver==1.2.4'
-PYTHON_CONFIGURATION='default@2.7.13.4f815b5:solaris10@2.7.8.4f815b5'
+PYTHON_CONFIGURATION='default@2.7.13.25cf4cf:solaris11@2.7.13.c3d8a7d:solaris10@2.7.8.c3d8a7d:freebsd11@2.7.13.641bbdca'
+BASE_REQUIREMENTS='chevah-brink==0.63.4 paver==1.2.4'
```

Comment:
I think that in python-package you can publish `641bbdca` for all platforms and use a simplified configuration... so that we don't have separate versions for freebsd11 and solaris11

--------

the question is whether we want to have many published version in production and get a clean config vs publishing only required changes and have a more complicated config